docs/source/author/writing_philosophy.rst

```diff
@@ -1,2 +1,9 @@
 Writing Philosophy
 ##############################################################################
+
+- how we present material: 
+    - Chapters cover a key area of computer science 
+        - contain an intro page and several chapter sections that each cover a key sub area of the main chapter topic
+        - always end with 'the whole story' section which is a summary of the chapter and a further reading section which contains links/book references 
```

Comment:
Call them `The whole story!` and `Further reading` because that's what they're called consistently